ui/src/shared/components/RefreshingGraph.js

```diff
@@ -51,6 +52,7 @@ const RefreshingGraph = ({
 
   return (
     <RefreshingLineGraph
+      key={manualRefresh || 0}
```

Comment:
Let's use `defaultProps` for this.  Also, I think our approach deserves a comment.

Reply:
Good idea.